gateway-api/modules/payments/src/main/java/io/knotx/example/payment/handler/fragments/TaskContextHandlerFactory.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
This should be moved to [Knot.x Fragments](https://github.com/Knotx/knotx-fragments) repository.